pkg/kubectl/cmd/get.go

```diff
@@ -543,7 +543,7 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 func addOpenAPIPrintColumnFlags(cmd *cobra.Command) {
 	cmd.Flags().Bool(useOpenAPIPrintColumnFlagLabel, false, "If true, use x-kubernetes-print-column metadata (if present) from openapi schema for displaying a resource.")
 	// marking it deprecated so that it is hidden from usage/help text.
-	cmd.Flags().MarkDeprecated(useOpenAPIPrintColumnFlagLabel, "its an experimental feature.")
+	cmd.Flags().MarkDeprecated(useOpenAPIPrintColumnFlagLabel, "Its an experimental feature.")
```

Comment:
There is no strict policy. I prefer to push "Address comments" commit on-top of the old commits. This makes reviews in Github much easier as you see what actually changed. Then, when you get a LGTM, you squash all of those mini commits. Usually, the reviewer will tell you that as well. In general, if you know that nobody looks actively on your code right now or needs to see the diff from last review, a force push is fine.